compliance/collection/nodeinventorizer/cachedInventorizer.go

```diff
@@ -0,0 +1,163 @@
+package nodeinventorizer
```

Comment:
Maybe, as outlined here https://github.com/stackrox/stackrox/pull/4701#discussion_r1114520018, it might be the better way to actually do what the location hints: building a `CachingInventorizer`. Let me try this, as it's a small change